internal/cmd/kafka/command_cluster_update.go

```diff
@@ -104,18 +178,17 @@ func (c *clusterCommand) validateResize(cku int32, currentCluster *cmkv2.CmkV2Cl
 	if currentCluster.Spec.GetAvailability() == highAvailability && cku <= 1 {
 		return 0, errors.New(errors.CKUMoreThanOneErrorMsg)
 	}
+
 	if cku == 0 {
 		return 0, errors.New(errors.CKUMoreThanZeroErrorMsg)
 	}
-	// Cluster can't be resized while it's provisioning or being expanded already.
```

Comment:
Is there a reason why we're removing these comments? Seems like it could be helpful.

Reply:
Those comments should've been cleaned up earlier. This function no longer checks if the cluster is provisioning or expanding already.